pkg/trace/stats/concentrator_test.go

```diff
@@ -197,18 +276,22 @@ func TestConcentratorStatsTotals(t *testing.T) {
 		}
 
 		for key, count := range stats[0].Counts {
-			if key == "query|duration|env:none,resource:resource1,service:A1" {
+			if strings.Contains(key, "|duration|") {
 				duration += count.Value
 			}
-			if key == "query|hits|env:none,resource:resource1,service:A1" {
```

Comment:
The pattern didn't scale well for additional cases, but since those cases have been removed we can change it back :+1:

I'd like to keep the tracking and assertion on errors though, since that was missing before.